setup.py

```diff
@@ -49,8 +49,26 @@
     ]},
     install_requires=['pytz'],
     extras_require={
-        "bcrypt": ["bcrypt"],
-        "argon2": ["argon2-cffi >= 16.1.0"],
+        'admindocs': ['docutils'],
+        'argon2': ['argon2_cffi>=16.1.0'],
+        'bcrypt': ['bcrypt'],
+        'gis': [
+            'geoip2',
+            'numpy',
+        ],
+        'geoip': ['geoip2'],
+        'jinja2': ['Jinja2>=2.9.2'],
+        'memcached': ['python3-memcached'],
```

Comment:
1. Given that we have `pylibmc` below, is `memcached` the right key here?
2. Given that there are choices here, is this something we want to have an opinion on? (It doesn't seem as clear to me as e.g. databases.)